src/components/Pagination/Pagination.tsx

```diff
@@ -1035,12 +1039,15 @@ class Pagination extends React.PureComponent<IProps, IState> {
     );
 
     return (
+      <>
+      <VisuallyHidden componentId="pagination_wrap">Pagination</VisuallyHidden>
       <ul
         className={classNames(theme['rc-pagination'], className)}
         style={props.style}
         unselectable="unselectable"
         ref={this.savePaginationNode}
         {...dataOrAriaAttributeProps}
+        aria-labeledby="pagination_wrap"
```

Comment:
@Farhan-emgage this property is misspelled. The correct spelling is "aria-labelledby".

Reply:
@weegrom  ok